docs/adr/stac-generation-ingestion-user-stories.md

```diff
@@ -0,0 +1,193 @@
+# MAAP STAC Metadata Generation and Ingestion User Stories
+
+This document captures the user stories for all use cases of the MAAP STAC metadata generation and ingestion pipeline (public and user-generated STACs).
+
+**Goal:** enumerate all common use-cases so that we can confidently design modular infrastructure that can serve all use-cases.
+
+## Use Case 1: DPS Job Outputs Cataloging in User-STAC
+
+### User Story 1A: Single DPS Job Output Cataloging
+
+**As a** researcher running a data processing job,  
+**I want to** have the outputs of my DPS job automatically cataloged in the user-STAC,  
+**So that** I can easily discover, access, visualize, and share my processed data products.
+
+**Acceptance Criteria:**
+
+- I can specify a collection ID (new or existing) for my job outputs
+- If I don't specify a collection, items are added to my default username-based collection
```

Comment:
Is it 1 job = 1 item?

Reply:
I have been thinking about this as one item per job, but I guess it is possible that a user would want to produce multiple STAC items in a STAC job. How common is that?

This is a case where producing full catalog/collection/item documents could help but I am wary about placing that technical burden on users.